internal/hcsoci/hcsdoc_wcow.go

```diff
@@ -77,6 +78,50 @@ func createMountsConfig(ctx context.Context, coi *createOptionsInternal) (*mount
 	return &config, nil
 }
 
+// HandleCPULimits handles the logic of converting and validating the CPU limits specified
+// in the OCI spec.
+func HandleCPULimits(ctx context.Context, cid, vmid string, spec *specs.Spec, hostCPUCount, uvmCPUCount int32) (int32, int32, int32, error) {
```

Comment:
The return values are pretty unclear here. Can we document or use named return values?

Reply:
`Handle` is also fairly generic and doesn't really describe what's being done. I would suggest a more descriptive name like `GetCPULimits` or `ConvertCPULimits`.